src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days <= 0) {
+    return 0;
+  }
+
+  if (days < 3) {
+    return days * 40;
+  } else if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  } else if (days < 7) {
```
Одну з умов можна прибрати. Адже якщо дійшло до цього else if, значить перша перевірка не пройшла, і днів точно не менше 3